ee/modules/030-cloud-provider-openstack/docs/CONFIGURATION.md

```diff
@@ -8,117 +8,4 @@ You can configure the number and parameters of ordering machines in the cloud vi
 
 ## Parameters
 
-The module settings are set automatically based on the placement strategy chosen. In most cases, you do not have to configure the module manually.
-
-If you need to configure a module because, say, you have a bare metal cluster and you need to enable additional instances from vSphere, then refer to the [How to configure a Hybrid cluster in vSphere](faq.html#how-do-i-create-a-hybrid-cluster) section.
-
-> **Note (!)** that if the parameters provided below are changed (i.e., the parameters specified in the deckhouse ConfigMap), the **existing Machines are NOT redeployed** (new machines will be created with the updated parameters). Redeployment is only performed when `NodeGroup` and `OpenStackInstanceClass` parameters are changed. You can learn more in the [node-manager](../../modules/040-node-manager/faq.html#how-do-i-redeploy-ephemeral-machines-in-the-cloud-with-a-new-configuration). module's documentation.
```

Comment:
You do not need to delete these comments. Let's return them.

Reply:
+